src/App.module.css

```diff
@@ -5,7 +5,8 @@
     margin: 0;
     display: grid;
     grid-template-columns: 25rem auto auto;
-    grid-template-rows: 100%;
+    /* todo: adjust bottom pane height, make foldable and/or responsive */
+    grid-template-rows: auto 10rem;
```

Comment:
@Janekdererste Can you help me here? I think we want to be able to hide/show the details using some button. Also the current css is not working very well as the details are moved to far down when we open the instructions list. For mobile I think we should hide the details entirely for now?!

Reply:
> For mobile I think we should hide the details entirely for now?!

I think this is done in https://github.com/graphhopper/graphhopper-maps/pull/74/files#diff-f5600dbc5cd82f7decd67da5f626d0205081d6a12e6da6ba1a5cbc14ea116c28R35

Reply:
After the recent merge the heightgraph is no longer shown for small screens